internal/controllers/vm_controller.go

```diff
@@ -91,7 +97,12 @@ func (r *VmReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Re
 	return ctrl.Result{}, nil
 }
 
-func (r *VmReconciler) setVmVolumesMetrics(ctx context.Context, vm *kubevirtv1.VirtualMachine) error {
+func getVmCrd() string {
+	vmKind := strings.ToLower(kubevirtv1.VirtualMachineGroupVersionKind.Kind) + "s"
```

Comment:
Same as `kubevirtv1.VirtualMachineGroupVersionKind.GroupKind().ToString()`?

Reply:
No, because there is the `s` at the end for plural. The CRD is called:
```
virtualmachines.kubevirt.io
```

And `kubevirtv1.VirtualMachineGroupVersionKind.GroupKind().ToString()` returns:
```
virtualmachine.kubevirt.io
```

Reply:
Silly question, but does using the plural make a difference?